src/rules/dollar-variable-colon-newline-after/index.js

```diff
@@ -0,0 +1,62 @@
+import {
+  declarationValueIndex,
+  namespace,
+  whitespaceChecker,
+} from "../../utils"
+import { utils } from "stylelint"
+
+export const ruleName = namespace("dollar-variable-colon-newline-after")
+
+export const messages = utils.ruleMessages(ruleName, {
+  expectedAfter: () => "Expected newline after \":\"",
+  rejectedAfter: () => "Unexpected whitespace after \":\"",
+  expectedAfterMultiLine: () => "Expected newline after \":\" with a multi-line value",
+})
+
+export default function (expectation) {
+  const checker = whitespaceChecker("newline", expectation, messages)
+  return (root, result) => {
+    const validOptions = utils.validateOptions(result, ruleName, {
+      actual: expectation,
+      possible: [
+        "always",
+        "always-multi-line",
+      ],
+    })
+    if (!validOptions) { return }
+
+    root.walkDecls(decl => {
+      if (!decl.prop || decl.prop[0] !== "$") { return }
```

Comment:
I'm not really sure. Such check won't be sufficient for cases like [this](https://github.com/kristerkari/stylelint-scss/blob/master/src/rules/media-feature-value-dollar-variable/index.js#L25) where the check for whether it's a $-variable has to be more thorough. Won't the proposed util name be misguiding?